docs/en/latest/tutorials/add-multiple-api-versions.md

```diff
@@ -0,0 +1,227 @@
+
+```shell
+http://org.apisix/hello?version=1
+http://org.apisix/hello?version=2
+```
+
+#### Custom Request Header
+
+You can also set the version number using custom headers in requests and responses. This leaves the URI of your resources unchanged.
+
+```shell
+http://org.apisix/hello -H 'Version: 1'
+http://org.apisix/hello -H 'Version: 2'
+```
+
+The primary goal of versioning is to provide users of an API with the most functionality possible while causing minimal inconvenience. Keeping this goal in mind, let’s have a look in this tutorial at how to _publish and manage multiple versions of your API_ with Apache APISIX.
+
+**In this tutorial**, you learn how to:
+
+- Create a route and upstream for our sample API.
+- Add a new version to the existing API.
+- Use [proxy-rewrite](https://apisix.apache.org/docs/apisix/plugins/proxy-rewrite/) plugin to rewrite the path in a plugin configuration.
+- Route API requests from the old version to the new one.
+
+## Prerequisites
+
+For the demo case, we will leverage the sample repository [Evolve APIs](https://github.com/nfrankel/evolve-apis) on GitHub built on the Spring boot that demonstrates our API. You can see the complete source code there.
+
+To execute and customize the example project per your need shown in this tutorial, here are the minimum requirements you need to install in your system:
+
+- [Docker Desktop](https://docs.docker.com/desktop/windows/install/) - you need [Docker desktop](https://www.docker.com/products/docker-desktop/) installed locally to complete this tutorial. It is available for [Windows](https://desktop.docker.com/win/edge/Docker%20Desktop%20Installer.exe) or [macOS](https://desktop.docker.com/mac/edge/Docker.dmg).
+
+Also, complete the following steps to run the sample project with Docker.
+
+Use [git](https://git-scm.com/downloads) to clone the repository:
+
+``` shell
+git clone 'https://github.com/nfrankel/evolve-apis'
+```
+
+Go to root directory of _evolve-apis_
+
+``` shell
+cd evolve-apis
+```
+
+Now we can start our application by running `docker compose up` command from the root folder of the project:
+
+``` shell
+docker compose up -d
+```
+
+### Create a route and upstream for the API.
+
+You first need to [route](https://apisix.apache.org/docs/apisix/terminology/route/) your HTTP requests from the gateway to an [upstream](https://apisix.apache.org/docs/apisix/terminology/upstream/) (your API). With APISIX, you can create a route by sending an HTTP request to the gateway.
+
+```shell
+curl http://apisix:9080/apisix/admin/routes/1 -H 'X-API-KEY: xyz' -X PUT -d '
+{
+  "name": "Direct Route to Old API",
+  "methods": ["GET"],
+  "uris": ["/hello", "/hello/", "/hello/*"],
+  "upstream": {
+    "type": "roundrobin",
+    "nodes": {
+      "oldapi:8081": 1
+    }
+  }
+}'
+```
+
+At this stage, we do not have yet any version and you can query the gateway as below:
+
+```shell
+> curl http://org.apisix/hello
```

Comment:
We can remove the `>` and probably split the command and the output. You can add the title "output" to the output block.

`shell title="output"`

Reply:
Good suggestion! Fixed it. Thanks!